src/components/Payroll/ConfirmWireDetails/ConfirmWireDetails.tsx

```diff
@@ -59,7 +59,7 @@ function Root({ companyId, wireInId, onEvent }: ConfirmWireDetailsProps) {
           modalContainerRef,
         }),
       ),
-    [companyId, wireInId, selectedWireInId],
+    [companyId, selectedWireInId],
```

Comment:
We should keep wireInId on this one


```suggestion
    [companyId, wireInId, selectedWireInId],
```